source/SIL.AppBuilder.Portal.Frontend/src/lib/toast.ts

```diff
@@ -13,11 +13,11 @@ export interface NotifyOptions {
   };
 }
 export function show(msg: string, options: NotifyOptions) {
-  notify.show(msg, options.type, options.timeout);
+  // notify.show(msg, options.type, options.timeout);
```

Comment:
commented these out for now, because we eventually are going to be using our own toast notifications, and not this library.

Reply:
Can this wait until you include own toast notification? When there are errors in DWKit, it is useful to see them.

Reply:
Well, I guess you left the error ones in place.  Still, it is useful for the moment to see messages from DWKit.

Reply:
DWKit doesn't use these ?